.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+updates:
+- package-ecosystem: npm
```

Comment:
We are using `yarn` but based on the [documentation's table](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#package-ecosystem) this setting should be `npm` .